roles/srv-announcer/tasks/install.yml

```diff
@@ -1,47 +1,23 @@
-- name: copying {{ srv_announcer_service_name }} artifact
-  copy:
-    src: "{{ sft_artifact_file_path }}"
-    dest: "{{ srv_announcer_artifact_target_path }}"
-  when:
-    - srv_announcer_artifact_file_path is defined
-    - srv_announcer_version is not defined
-    - srv_announcer_artifact_file_url is not defined
-    - srv_announcer_artifact_checksum is not defined
-
-- name: downloading {{ srv_announcer_service_name }} artifact by version
-  get_url:
-    url: "{{ srv_announcer_download_base_url }}/{{ srv_announcer_artifact_basename }}.tar.gz"
-    checksum: "sha256:{{ srv_announcer_versions[ srv_announcer_version ][ srv_announcer_platform ~ '_' ~ srv_announcer_arch ] }}"
-    dest: "{{ srv_announcer_artifact_target_path }}"
-  when:
-    - srv_announcer_version is defined
-    - srv_announcer_artifact_file_path is not defined
-    - srv_announcer_artifact_file_url is not defined
-    - srv_announcer_artifact_checksum is not defined
-
 - name: downloading {{ srv_announcer_service_name }} artifact by URL
   get_url:
     url: "{{ srv_announcer_artifact_file_url }}"
-    checksum: "sha256:{{ srv_announcer_artifact_checksum }}"
-    dest: "{{ srv_announcer_artifact_target_path }}"
+    checksum: "{{ srv_announcer_artifact_checksum }}"
+    dest: "/tmp/srv-announcer.tar.gz"
```

Comment:
question: what is the reasoning behind moving away from using the existing constant `srv_announcer_artifact_target_path`?

Reply:
Strictly speaking, even this is wrong, and we shouldn't drop files in before-known temporary folders. See https://cwe.mitre.org/data/definitions/377.html.

We should either create a temporary folder to download and unpack the artifact (using the `tempfile` module), or just serve the (single) binary directly.

Reply:
Would you be fine if we'd serve the file directly?

Reply:
I dont see a reason for not serving an archive. Being able to provide some sort of manual/information alongside the binary is useful. I'll bring up a new commit to this branch that makes use of the `tempfile` task. Feel free to revert it if it's not to your liking.

Reply:
Sounds good to me